src/zh/2017-03/info.yml

```diff
@@ -0,0 +1,6 @@
+---
+  title: "来自加拉太书的福音"
+  description: "本季的研经指引邀请我们与使徒保罗一起，经历他恳劝加拉太信徒忠于基督的心路历程。与此同时，我们也有机会深思自己对于真理的理解，这真理曾使得马丁·路德最终与罗马教廷决裂，并恢复了《圣经》中的福音。"
+  human_date: "2017年第3季"
+  start_date: "24/06/2017"
+  end_date: "29/09/2017"
```

Comment:
@imasaru You need to provide colors here, otherwise apps will be crashing. I will update `create.js` script to do it automatically later.